base/base.go

```diff
@@ -159,6 +159,11 @@ func (s *Server) listenForMsgs(shutdownCh chan struct{}, sub *kbchat.NewSubscrip
 					s.Errorf("listenForMsgs: unable to handlePProf: %v", err)
 				}
 				continue
+			case strings.HasPrefix(cmd, "!feedback-"):
```

Comment:
why the dash?

Reply:
oh i see, there's a suffix. nvm!

Reply:
I think the bots each have a suffix (ie. `!feedback-githubbot`) so that `!feedback` doesn't overlap if there are many managed bots in one channel.

Reply:
to match https://github.com/keybase/managed-bots/pull/107/files#diff-f494338fe77eff0d93e32db8d4b11c8cR310 and prevent a clash with a legit `feedback` command. Maybe we should make botlog/logsend/pprof prefixed with ` _` or something